src/models/user.js

```diff
@@ -58,5 +70,17 @@ export default {
         favoriteResults: [...state.favoriteResults, payload],
       };
     },
+    removeFavoriteController(state, { payload }) {
+      return {
+        ...state,
+        favoriteControllers: state.favoriteControllers.filter(item => item.key !== payload.key),
+      };
+    },
+    removeFavoriteResult(state, { payload }) {
+      return {
+        ...state,
+        favoriteResults: state.favoriteResults.filter(item => item.key !== payload.key),
+      };
+    },
```

Comment:
This isn't directly related to your changes, but I hadn't really thought about this before. It's interesting that "favorites" is just a transient Redux property, and not persisted across browser sessions. Should this be part of user metadata at some point?

Reply:
@dbutenhof Exactly, this is stored as in a "browser" storage, hence persisted across browser sessions. We also would like to have a functionality which lets users clean their local storage, which would delete all these records. To make it persist across "user" sessions, we would ideally save it in the backend smh. @portante Is there a need to do that in the near future?

Reply:
> We also would like to have a functionality which lets users clean their local storage, which would delete all these records.

Can you file an issue to track this?

> To make it persist across "user" sessions, we would ideally save it in the backend smh. @portante Is there a need to do that in the near future?

This will become a requirement once the notion of a user is implemented in both the Dashboard and in the backend server.